config/environments/development.rb

```diff
@@ -8,6 +8,7 @@
   # In the development environment your application's code is reloaded any time
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Set configs for https://github.com/ryanb/letter_opener, https://github.com/fgrehm/letter_opener_web
(fake mail server for development)